airflow/contrib/operators/hive_to_dynamodb.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from airflow.hooks.hive_hooks import HiveServer2Hook
+from airflow.contrib.hooks.aws_dynamodb_hook import AwsDynamoDBHook
+
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
+
+
+class HiveToDynamoDBTransfer(BaseOperator):
+    """
+    Moves data from Hive to DynamoDB, note that for now the data is loaded
+    into memory before being pushed to DynamoDB, so this operator should
+    be used for smallish amount of data.
+
+    :param sql: SQL query to execute against the hive database
+    :type sql: str
+    :param table_name: target DynamoDB table
+    :type table_name: str
+    :param table_keys: partition key and sort key
+    :type table_keys: list
+    :param aws_conn_id: aws connection
+    :type aws_conn_id: str
+    :param hiveserver2_conn_id: source hive connection
+    :type hiveserver2_conn_id: str
+    """
+
+    template_fields = ('sql')
```

Comment:
Is this ever used?

Reply:
@skudriashev in my understanding this might be used if we pass sql for instance:  select * from {{ ds }} where ds is a template field.

Reply:
@sid88in I don't see there you use this within the code.

Reply:
@sid88in you are right)

Reply:
@skudriashev as we discussed on gitter - template_fields is valid. I have replaced ('sql') by ('sql',) thanks